src/placement/jump_map.c

```diff
@@ -1184,10 +1185,9 @@ jump_map_obj_place(struct pl_map *map, struct daos_obj_md *md,
  *                              another target, Or 0 if none need to be rebuilt.
  */
 static int
-jump_map_obj_find_rebuild(struct pl_map *map, struct daos_obj_md *md,
-			  struct daos_obj_shard_md *shard_md,
-			  uint32_t rebuild_ver, uint32_t *tgt_id,
-			  uint32_t *shard_idx, unsigned int array_size)
+jump_map_obj_find_rebuild(struct pl_map *map, uint32_t gl_layout_ver, struct daos_obj_md *md,
```

Comment:
minor, may use the same name as "layout_gl_version" for the same argument

Reply:
sure